docs/source/commandline/commandline-all.rst

```diff
@@ -1931,33 +1931,63 @@ Core Input Options
 
 **-joystick_map** *<map>* / **-joymap** *<map>*
 
-    Controls how joystick values map to digital joystick controls. MAME accepts
-    all joystick input from the system as analog data.  For true analog
-    joysticks, this needs to be mapped down to the usual 4-way or 8-way digital
-    joystick values.  To do this, MAME divides the analog range into a 9x9 grid.
-    It then takes the joystick axis position (for X and Y axes only), maps it to
-    this grid, and then looks up a translation from a joystick map.  This
+    Controls how joystick values map to digital joystick controls.   
+    
+    Systems such as Pac-Man use a 4-way joystick and will actually exhibit
+    undesired behavior when a diagonal is triggered; in the case of Pac-Man,
+    movement will stop completely at intersections when diagonals are triggered
+    and the game will be considerably harder to play correctly.  Many other
+    arcade cabinets used 4-way or 8-way joysticks (as opposed to full analog
+    joysticks), so for PC-style true analog joysticks this then needs to be
+    mapped down to the expected 4-way or 8-way digital joystick values.  This
+    is because MAME accepts all joystick input from your controller as analog
+    data.
+    
+    To do this, MAME divides the analog range into a 9x9 grid that looks
+    like this:
+    
+    **insert 9x9 grid picture here**
+    
+    MAME then takes the joystick axis position (for X and Y axes only), maps it
+    to this grid, and then looks up a translation from a joystick map.  This
     parameter allows you to specify the map.
 
+    For instance, an 8-way joystick map traditionally looks like this:
+
+    **insert 8-way map picture here**
+
+    This mapping gives considerable leeway to the angles accepted for a given
+    direction, so that being approximately in the area of the direction you want
+    will give you the results you want.  Without that, if you were slightly off
+    center while holding the stick left, it would not recognize the action
+    correctly.
+
     The default is ``auto``, which means that a standard 8-way, 4-way, or 4-way
     diagonal map is selected automatically based on the input port configuration
     of the current system.
 
-    Maps are defined as a string of numbers and characters. Since the grid is
-    9x9, there are a total of 81 characters necessary to define a complete map.
-    Below is an example map for an 8-way joystick:
+    Generally you will want to set up the **-joystick_map** setting in the
+    per-system ``<system>.cfg`` file as opposed to the main ``MAME.INI``
```

Comment:
It doesn't go in the `.cfg` file – it goes in the system-specific `.ini` file.

Reply:
Fixed.